Makefile

```diff
@@ -0,0 +1,5 @@
+test: .bundle/vader.vim
```

Comment:
Here I might need to find a way to remove the build artifacts after running `make test` locally, though this isn't of high priority currently.

Reply:
Resolved in b3fce85 with a new target `clean`